routes/teachers.js

```diff
@@ -59,36 +59,38 @@ router.get('/site/:siteId', async (req, res) => {
   }
 });
 
-// create a teacher
+// create a teacher in firebase and in the backend
 router.post('/', async (req, res) => {
   try {
-    const { firebaseId, firstName, lastName, phoneNumber, email, inviteId, notes } = req.body;
-    isAlphaNumeric(firebaseId, 'Firebase Id must be alphanumeric');
-    isPhoneNumber(phoneNumber, 'Invalid Phone Number');
-    isNanoId(inviteId, 'Invalid Invite Id Format');
+    const { firstName, lastName, email, notes, password, siteIds } = req.body;
+
+    const newUser = await firebaseAdmin.auth().createUser({
+      email,
+      emailVerified: true,
+      password,
+    });
 
     const newTeacher = await pool.query(
       `INSERT INTO tlp_user
         (firebase_id, first_name, last_name,
-        phone_number, email, position, active, notes)
```

Comment:
Why did you remove phone number? It is still an optional field right?

Reply:
Also add an IF check to the PUT request to handle the phone number potentially being null

Reply:
The phone number was removed from teacher account creation as mentioned [here](https://ctcuci.slack.com/archives/C02JMK343V4/p1652903148531539?thread_ts=1652771107.951369&cid=C02JMK343V4)